deploy/travella-staging.ingress.yml.tpl

```diff
@@ -8,6 +8,8 @@ metadata:
   annotations:
     kubernetes.io/ingress.global-static-ip-name: {{ INGRESS_STATIC_IP_NAME }}
 spec:
+  tls:
+    - secretName: {{ PROJECT_NAME }}-tls-secrets
   rules:
   - host: travela-staging.andela.com
```

Comment:
It LGTM, however I am curious as to why for the most part the keyword used is `travella` with the double `L` however the domain name has just one `travela-staging.andela.com`

Reply:
These files were created before the name was reverted. The name cleanup will be done once the we infrastructure is stable.